awx/settings/defaults.py

```diff
@@ -1239,3 +1232,29 @@ def IS_TESTING(argv=None):
     'awx.main.middleware.URLModificationMiddleware',
     'awx.main.middleware.SessionTimeoutMiddleware',
 ]
+
+# Secret header value to exchange for websockets responsible for distributing websocket messages.
+# This needs to be kept secret and randomly generated
+BROADCAST_WEBSOCKET_SECRET = ''
+
+# Port for broadcast websockets to connect to
+# Note: that the clients will follow redirect responses
+BROADCAST_WEBSOCKET_PORT = 443
+
+# Whether or not broadcast websockets should check nginx certs when interconnecting
+BROADCAST_WEBSOCKET_VERIFY_CERT = False
```

Comment:
Should we default this to `True` in `production.py` ?

Reply:
Yes, I need to do the installer work to setup a self-signed certificate for nginx specifically for these websockets.